README.rst

```diff
@@ -195,9 +187,50 @@ to the name of the folder on the cloud (e.g. 'exp_data').
 To get set up with Rclone, download Rclone and follow the documentation which should include
 running the command ``rclone config`` enabling you to set up the location and name of the cloud
 drive you wish to pull from. The documentation of rclone is pretty straightforward and can walk
-you through this.
-Once your Rclone config file is set up open your ``~/_pyspecdata`` or ``~/.pyspecdata`` config file
-and edit the data_directory value pair to point to your RcloneRemotes.
+you through this. 
+
+Setting up your _pyspecdata configuration file
+----------------------------------------------
+Part of the pySpecData package is the datadir module, allowing the user to run the same code on 
+different machines - even thought he location of the raw spectral data might change. 
+This is controlled by the ``~/.pyspecdata`` or ``~/_pyspecdata`` config file. 
+On top should be the ``[General]`` section, e.g.
+
+::
+
+    [General]
+    data_directory = /home/jmfranck/exp_data
+    qesr conversion = 162.66
+    qesr diameter = 0.704
+    qesr q = 4700
+
+This section points to the directory with the datasets of interest whether that is the
+direct path to the drive with the datasets or if you prefer Rclone, this ``data_directory``
+points to your local folder of datasets.
+
+Following the ``General`` section is ``ExpTypes`` which gives the various locations to 
+folders containing the appropriate data sets. For example,
+
+::
+
+    [ExpTypes]
+    odnp_nmr_comp/odnp = /home/jmfranck/exp_data/NMR_comp/ODNP
+
+gives the location of the ODNP datasets. So when you call ``odnp_nmr_comp/odnp`` this will point
+to the actual location, ``/home/jmfranck/exp_data/NMR_comp/ODNP``
+
+Finally add in the ``RcloneRemote`` key to your config file:
+
+::
```

Comment:
please gather the file into a single file, and then explain underneath.  It's easy to refer to the sections

Reply:
most of these sections are created automatically -- you're not telling them how to write a config file from scratch -- just what it means, so that they can edit it

Reply:
okay condensed literal block and discuss what each section is